.vitepress/config.ts

```diff
@@ -122,31 +122,28 @@ export default defineConfigWithTheme<ThemeConfig>({
     template: {
       compilerOptions: {
         isCustomElement: (tag) => {
-          return ["chatlio-widget"].includes(tag.toLowerCase());
-        }
-      }
-    }
+          return ['chatlio-widget'].includes(tag.toLowerCase());
+        },
+      },
+    },
   },
 });
 
 function getAnalyticsTags(env: NodeJS.ProcessEnv): HeadConfig[] {
-  if (!env.VITE_GTM_ID) {
-    return [];
+  const gtmIdPattern = /^GTM-[A-Z\d]+$/;
+  const gtmId = (env.VITE_GTM_ID || '').trim();
+  if (gtmId && gtmIdPattern.test(gtmId)) {
```

Comment:
We check that the `VITE_GTM_ID` environment variable uses the `GTM-` prefix (note that before this change we were using a Google Analytics property id starting with `G-` instead).